### ISSUE TITLE
Create Windows compatible repository to avoid 'invalid path' on git clone

### ISSUE THREAD

**Issue by unapproachable** [ivandokov/phockup#101]
Trying to clone the repository to Windows yields checkout errors from git due to the 'bad' filename used for testing.  Is there a way to have a filesystem compatible test that doesn't break the repo on clone for Windows users?

```shell
P:\Projects>git clone https://github.com/ivandokov/phockup.git
Cloning into 'phockup'...
remote: Enumerating objects: 173, done.
remote: Counting objects: 100% (173/173), done.
remote: Compressing objects: 100% (103/103), done.
remote: Total 938 (delta 85), reused 125 (delta 43), pack-reused 765R
Receiving objects: 100% (938/938), 167.19 KiB | 2.79 MiB/s, done.
Resolving deltas: 100% (553/553), done.
error: invalid path 'tests/input/!#$%&'"*+-.^_`|~:.jpg'
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'
```
For myself and others to move forward, Windows users can do the following after clone:

```shell
P:\Projects>cd phockup
P:\Projects\phockup>git config core.protectNTFS false
P:\Projects\phockup>git checkout master
error: unable to create file tests/input/!#$%&'"*+-.^_`|~:.jpg: Invalid argument
Already on 'master'
D       "tests/input/!#$%&'\"*+-.^_`|~:.jpg"
Your branch is up to date with 'origin/master'.
```

Which has the unfortunate consequence of producing this on future updates:

```shell
P:\Projects\phockup>git pull
error: cannot pull with rebase: You have unstaged changes.
error: please commit or stash them.
```

Where the unstaged change is the deleted invalid path from the original checkout.

On Linux it's obviously fine:

```shell
$ git clone https://github.com/ivandokov/phockup.git
Cloning into 'phockup'...
remote: Enumerating objects: 173, done.
remote: Counting objects: 100% (173/173), done.
remote: Compressing objects: 100% (103/103), done.
remote: Total 938 (delta 85), reused 125 (delta 43), pack-reused 765
Receiving objects: 100% (938/938), 167.19 KiB | 0 bytes/s, done.
Resolving deltas: 100% (553/553), done.
Checking connectivity... done.
```

**Comment by ivandokov** [comment]
To be honest I do not have Windows development machine and I am really not sure what filename will fit into the allowed filenames for Windows and still be useable for the test.

**Comment by unapproachable** [comment]
Can you clarify what the test is doing?  It looks like it's an exiftool test more than a phockup test.  Is the intent to create a filename that has invalid characters for the exiftool or for the host filesystem?  Would it be possible to create the file on the target system as part of the test rather than include it in the repository?  I'd be happy to help with a PR, but I'd need to better understand how to work around this filename for testing purposes.

**Comment by ivandokov** [comment]
The issue that raised the question for "illegal" characters in file names is #52 and the PR that fixed the wrong behaviour is #53 but I have to admit the test is kind of extreme and the file name used for testing this behaviour can be "toned down" to fit in Windows requirements. 
PR is more than welcome! Thank you @unapproachable.
